packages/backend/convex/crons.ts

```diff
@@ -0,0 +1,16 @@
+import { cronJobs } from "convex/server";
+import { internal } from "./_generated/api";
+
+const crons = cronJobs();
+
+// Run cleanup of old queued matches every 2 minutes
+// This prevents matches from staying in "Queuing" status indefinitely
+crons.interval(
+  "archive old queued matches",
+  { minutes: 2 },
```

Comment:
Let's increase the timeout. 
```suggestion
  { minutes: 10 },
```